scripts/split_repo_migration/delete-everything-split-repo.sh

```diff
@@ -0,0 +1,145 @@
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script was written as part of a project to migrate
+# split repositories to a mono repository google-cloud-python.
+# This script will delete all files in a split repository and
+# update the README to indicate that the repository is archived.
+
+# sourced vs execution detection obtained from https://stackoverflow.com/a/28776166
+SOURCED=0
+if [ -n "$ZSH_VERSION" ]; then
+  case $ZSH_EVAL_CONTEXT in *:file) SOURCED=1;; esac
+elif [ -n "$KSH_VERSION" ]; then
+  [ "$(cd -- "$(dirname -- "$0")" && pwd -P)/$(basename -- "$0")" != "$(cd -- "$(dirname -- "${.sh.file}")" && pwd -P)/$(basename -- "${.sh.file}")" ] && SOURCED=1
+elif [ -n "$BASH_VERSION" ]; then
+  (return 0 2>/dev/null) && SOURCED=1
+else # All other shells: examine $0 for known shell binary filenames.
+     # Detects `sh` and `dash`; add additional shell filenames as needed.
+  case ${0##*/} in sh|-sh|dash|-dash) SOURCED=1;; esac
+fi
+
+(( SOURCED != 1 )) || { \
+  echo "Please do not source this script, but execute it directly."
+  return -10
+}
+
+# We require executing the script so that an early exit (explicitly or via -e)
+# does not kill the user's shell.
+
+# `-e` enables the script to automatically fail when a command fails
+set -e
+
+# Ensure that both arguments are provided
+if [ $# -lt 2 ]
+then
+  echo "Usage: $0 <split-repo-name> <target-path>
+
+Args:
+   split-repo-name: The name of the split repository. For example, \`python-access-approval\`.
+   artifact-name: The name of the artifact on PyPI. For example, \`google-cloud-access-approval\`.
+"
+  exit 1
+fi
+
+# The name of the split repository. (e.g. python-access-approval)
+SPLIT_REPO=$1
+# The name of the artifact on PyPI. (e.g. google-cloud-access-approval)
+ARTIFACT_NAME=$2
+
+SPLIT_REPO_DIR="/tmp/delete-library.${SPLIT_REPO}.$(date +"%Y-%m-%d.%H:%M:%S")"
+
+git clone "git@github.com:googleapis/${SPLIT_REPO}.git" $SPLIT_REPO_DIR
+
+cd $SPLIT_REPO_DIR
+
+# Create a git branch
+git checkout -b 'migrate-library'
+
+# Quick check to make sure there are no handwritten samples. 
+# Handwritten samples should be migrated to python-docs-samples before repository code is deleted.
+# Exclude the samples/generated_samples/* directory which contains autogenerated samples.
+NUM_PY_SAMPLES_FILES=$(find samples -type f -name "*.py" ! -path "samples/generated_samples/*" | grep -v "nox.*\.py" | grep -v "__init__\.py" | wc -l)
+
+# Fail if there are handwritten samples.
+if [[ $NUM_PY_SAMPLES_FILES -ne "0" ]]; then
+  echo "Please migrate handwritten samples to python-docs-samples before deleting the source."
+  exit 1
+fi
+
+# Fail if this repository has a customized version of the autogenerated code.
+# This specific check is to ensure that autogenerated files are not excluded.
+STANDARD_EXCLUDES=$(grep owlbot.py -e 'excludes=\["\*\*/gapic_version.py"]' | wc -l)
+
+if [[ $STANDARD_EXCLUDES != "1" ]]; then
```

Comment:
consider `-ne` here and below

Reply:
Fixed in [c19dfe4](https://github.com/googleapis/google-cloud-python/pull/10879/commits/c19dfe44d4ee3be762702d2c5cbc936c9c600800)